webapp/channels/src/components/integrations/__snapshots__/abstract_outgoing_webhook.test.tsx.snap

```diff
@@ -6,16 +6,16 @@ exports[`components/integrations/AbstractOutgoingWebhook should match snapshot 1
 >
   <BackstageHeader>
     <Link
-      to="/test/integrations/outgoing_webhooks"
```

Comment:
**Note:** All of the changes to the snapshot here should be due to changes to the props down in `webapp/channels/src/components/integrations/abstract_outgoing_webhook.test.tsx`. Please let me know if there are specific considerations around snapshots.